2-dom-manipulation/form.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const VAL_EMAIL = /^(([^<>()[\]\.,;:\s@\"]+(\.[^<>()[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+
+function validateForm(event){
+  event.preventDefault();
+
+  let name = document.querySelector('#contact_form').querySelector('#user_name'),  
+  email = document.querySelector('#contact_form').querySelector('#email_this'),
+  number = document.querySelector('#contact_form').querySelector('#user_phone_number'),
+  textBox = document.querySelector('#contact_form').querySelector('#text-box'),  
+  box =  document.querySelector('#contact_form').querySelectorAll('input[type="checkbox"]');
```

Comment:
you can save the form reference so you get the elements faster. It is also better to use const since these values will never change. Grouping multiple variable definitions is not easy to read, so it's better if you keep them separated.

```suggestion
 const form =  document.querySelector('#contact_form');
  const name = form.querySelector('#user_name'); 
  const email = form.querySelector('#email_this');
  const number = form.querySelector('#user_phone_number');
  const textBox = form.querySelector('#text-box');
  const box =  form.querySelectorAll('input[type="checkbox"]');
```